data/map planets.txt

```diff
@@ -2649,6 +2708,15 @@ planet Memory
 		threshold 4000
 		fleet "Small Deep Security" 16
 
+planet "Mendez Station"
+	attributes CCOR station
+	landscape land/station20
+	description 'Mendez Station is the closest object to Alpherg's twin stars. Mostly uninhabited Mendez sits in low orbit using it's monumental ramscoops to collect solar wind to process into fuel. The station is uncomfortably hot, life support systems only turning on and beginning to cool the habitation segment when a ship is docked. The refining process here is almost entirely automated, the habitation segment was only expected to be in use during routine inspections and maintenance.'
+	spaceport 'Forgoing a traditional docking ring, which would be excessive for such a low traffic station, Mendez's 'spaceport' is a small number of almost archaic docking arms leading to the hab segment. There was a security system built into each airlock, engraved with the seal of the CCOR government, but they've all been either smashed, bypassed, or hacked allowing unrestricted access to the station.'
```

Comment:
```suggestion
	spaceport `Forgoing a traditional docking ring - excessive for such a low traffic station - Mendez's "spaceport" consists of a small number of archaic docking arms leading to the habitation segment. There was a security system built into the airlocks, each engraved with the seal of the CCOR government, but they've all been either smashed, bypassed, or hacked, allowing unrestricted access to the station.`
```
The run-on sentence in the second sentence annoys me, but I can't think of any way to make it better?

Reply:
Resolving since the station no longer has a spaceport.